.changeset/spicy-ants-melt.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Instead of this empty changeset we can use our admin perms to merge even without the changesets CI passing